data-pipeline/transformers/test_normalizer.py

```
@@ -0,0 +1,53 @@
+            "unit": "mg/L",
+            "measurement_date": "2026-03-11",
+            "source": "government"
+        }
+        normalized = self.normalizer.normalize(raw_record, source="jal_shakti")
+        
+        self.assertEqual(normalized['parameter'], "DO")
+        self.assertEqual(normalized['value'], 7.2)
+
+    def test_parameter_mapping(self):
+        # Test fuzzy parameter mapping
+        self.assertEqual(self.normalizer.map_parameter("Biochemical Oxygen Demand"), "BOD")
```

Ruff (0.15.5)

[warning] Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)